templates/default/nginx-proxy-ssl-only.erb

```diff
@@ -0,0 +1,33 @@
+server {
+    listen 443 default_server ssl;
```

Comment:
It's pretty simple to listen on :80 + redirect to https too. See `elk-nginx-proxy-conf.erb`